content/en/storage-providers/seal-workers/post-workers.md

```diff
@@ -0,0 +1,155 @@
+
+### Minminal spec requirements
+
+These are the minimal requirements for running each of the PoSt tasks:
+
+| **Task**    | **GPU**            | **VRAM** | **RAM** |
+|:-----------:|:------------------:|:--------:|:-------:|
+| winningPoSt | Highly recommended |          |         |
+| windowPoSt  | Highly recommended | 8\.5GiB  | 128GiB  |
+
+{{< alert icon="callout" >}}
+Although both tasks can be run with a powerful CPU, it is highly recommended to run on GPU as it speeds up the process significantly, therefore reducing the chance to miss windowPoSt or winningPoSt
+{{< /alert >}}
+
+### Remote storage access
+
+The windowPoSt process requires reading random leafs of all the sealed sectors in a proving deadline. When setting up windowPoSt workers one needs to consider how the workers can access those files. The PoSt workers can ask any other worker to read challenges for them, including the lotus-miner process, but it will prefer reading it from local paths.
+
+{{< alert icon="warning" >}}
+Storage providers should design their worker sectors access according to their setup size and also redundancy required. If only one worker has read access to the sealed sectors, it can create a single point of failure. If you have multiple partitions in a deadline, having multiple PoSt workers requesting a single worker to read challenges for them may cause unwanted I/O load.
+{{< /alert >}}
+
+### Environment variables
+
+Remember to have the appropriate Nvidia-drivers and nvidia-opencl-icd installed if running OpenCL on your worker. If using CUDA, install the CUDA-toolkit and build lotus binaries with `FFI_USE_CUDA=1`
+
+The following environment variables are required to be set before starting the worker. These variables are similar to those set on the lotus-miner node as [explained in the setup guide]({{< relref "configure#setup" >}}):
+
+```
+export MINER_API_INFO:<TOKEN>:/ip4/<miner_api_address>/tcp/<port>/http
+export BELLMAN_CUSTOM_GPU="MODEL-NAME:CORES" # If you´re using a custom GPU
+export FIL_PROOFS_PARAMETER_CACHE=/fast/disk/folder # > 100GiB!
+export FIL_PROOFS_PARENT_CACHE=/fast/disk/folder2   # > 50GiB!
+```
+
+When the PoSt worker starts, it needs to read and verify the Filecoin proof parameters. We recommend copying them over from your lotus-miner machine. Otherwise, they will be downloaded locally on first run.
+
+The PoSt workers will fail to start if the file descriptor limit is not set high enough. This limit can be raised temporarily before starting the worker by running the command `ulimit -n 1048576`. Although, we recommend setting it permanently by following the [Permanently Setting Your ULIMIT System Value]({{< relref "kb#soft-fd-limit" >}}) guide.
```

Comment:
As of https://github.com/filecoin-project/lotus/pull/8390 all workers (sealing / post) require at least 100k fd limit, so we probably want to add this note to sealing worker page as well

Reply:
Nice! Will add it to that page as well.